pages/index.jsx

```diff
@@ -15,17 +16,25 @@ export default function Home() {
             <AppLayout>
                 <Placeholder title='Hero' />
                 <Placeholder id='about' title='About Us' />
-                <Banner
-                    title={labels.ourMission.title}
-                    description={labels.ourMission.description}
-                />
-                <Banner title={labels.ourVision.title} description={labels.ourVision.description} />
+                <div className={styles.logoOnmission}>
```

Comment:
This is a background image, should be set by CSS to the `<main>` tag

Reply:
I put it in the Banner.module.css. Is that what you meant by main?

Reply:
This one :) https://github.com/brito-ca/brito-website/blob/eea08c6fdf30a2539faa7f62361251a21a04326c/components/AppLayout/AppLayout.jsx#L11

Reply:
ok, thanks. Done.